src/test/java/com/auth0/client/MockServer.java

```diff
@@ -117,7 +119,7 @@ public void jsonResponse(String path, int statusCode) throws IOException {
         server.enqueue(response);
     }
 
-    public void rateLimitReachedResponse(long limit, long remaining, long reset) throws IOException {
```

Comment:
linter pointed out exceptions not thrown, and auto-closing of buffer (with jdk 8 try/catch)